### DISCUSSION TITLE
Decision tree

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/MLAlgorithms#23] [Q&A] [answered]
What's entropy?

**Comment by Mazin7d4** [comment]
Entropy is a measure of the randomness or uncertainty in a dataset. It's used in building decision trees. 

High entropy means the data has a lot of randomness or uncertainty, while low entropy indicates more predictability.

When building decision trees, splits are chosen to reduce entropy, aiming to gain clear and distinct divisions in the data.

**Comment by Haarika78** [comment, marked as answer]
Entropy is the measurement of disorder or impurities in the information processed in machine learning. It determines how a decision tree chooses to split data.
